src/Employee.java

```diff
@@ -0,0 +1,58 @@
+package Base;
+
+import java.util.Objects;
+
+public class Employee {
+    private String name;
+    private String department;
+    private double salary;
+    public static int count = 1;
+    private Integer id = 1;
+    public Employee() {
+        count++;
+    }
+    public Employee(String name, String department, double salary, Integer id) {
+        this.name = name;
+        this.department = department;
+        this.salary = salary;
+        this.id = id;
+    }
```

Comment:
Тут нужно немного изменить конструктор, чтобы id генерировался автоматически:
```suggestion
    public Employee(String name, int department, double salary) {
        this.name = name;
        this.department = department;
        this.salary = salary;
        this.id = count++;
    }

```